internal/cmd/shell.go

```diff
@@ -65,9 +66,10 @@ func shValidArgs(cmd *cobra.Command, args []string) error {
 
 func shCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:                "sh [environment_name] [<command [args...]>]",
-		Short:              "Open a shell and execute commands in a Coder environment",
-		Long:               "Execute a remote command on the environment\\nIf no command is specified, the default shell is opened.",
+		Use:   "sh [environment_name] [<command [args...]>]",
+		Short: "Open a shell and execute commands in a Coder environment",
+		Long: "Execute a remote command on the environment\\nIf no command is specified, the default shell is opened.\\n" +
```

Comment:
I think the docs will render properly if we use a real multi-line string here.

Reply:
It will yes, I can change it to a multiline. I dislike the text tabbed all the way left in my IDE ha.